packages/browser-wallet/src/popup/shared/IdCard/IdCard.tsx

```diff
@@ -100,12 +121,21 @@ export default function IdCard({ name, provider, status, onNameChange, className
             }}
         >
             <header className="id-card__header">
```

Comment:
![Screenshot from 2022-12-09 13-15-36](https://user-images.githubusercontent.com/87635671/206700785-2a30657b-5f77-4369-b37d-760cef54476d.png)
Sometimes there is space between the header and the border (I think I've also seen it with the old version, though)

Reply:
You're right, fixed this. It was due to a border-width being set with rem, which doesn't seem to be a good idea in cases like this.